command/agent_info.go

```diff
@@ -70,6 +90,18 @@ func (c *AgentInfoCommand) Run(args []string) int {
 		return 1
 	}
 
+	// If output format is specified, format and output the agent info
+	if json || len(tmpl) > 0 {
+		out, err := Format(json, tmpl, info)
+		if err != nil {
+			c.Ui.Error(err.Error())
```

Comment:
I think a `fmt.Sprintf` to add context around the error would be useful here

Reply:
Fixed